website/client/src/components/static/home.vue

```diff
@@ -909,6 +909,9 @@ export default {
       // windows: WINDOWS_CLIENT_ID,
       google: process.env.GOOGLE_CLIENT_ID, // eslint-disable-line
     });
+    this.$store.dispatch('common:setTitle', {
+      fullTitle: this.$t('socialTitle'),
```

Comment:
I can't find this string, also because Habitica - Gamify Your Life should't be changed I think you can set it directly in english so `fullTitle: 'Habitica - Gamify Your Life', `